jobs/inprogress_update/inprogress_update.py

```diff
@@ -74,56 +106,77 @@ def inprogress_update(user: User, max_rows: int, client_delay: int, examine_dela
             limit(max_rows). \
             with_for_update().all()
 
-        for r in examine_reqs:
+        for request in examine_reqs:
             row_count += 1
 
-            current_app.logger.debug(f'processing: {r.nrNum}')
-            current_app.logger.debug(f'nr {r.nrNum}, state: {r.stateCd} last_update:{r.lastUpdate}')
+            current_app.logger.debug(f'processing: {request.nrNum}')
+            current_app.logger.debug(f'nr {request.nrNum}, state: {request.stateCd} last_update:{request.lastUpdate}')
+
+            furnish_request_message(request, 'examiners-edit-inprogress')
 
             # if this NR was previously in DRAFT, reset it to that state
             # (ie: the user walked away from an open edit window)
             event = None
-            if r.previousStateCd == State.DRAFT:
-                r.stateCd = State.DRAFT
-                r.previousStateCd = None
+            if request.previousStateCd == State.DRAFT:
+                request.stateCd = State.DRAFT
+                request.previousStateCd = None
                 event = Event.SET_TO_DRAFT
             # otherwise put it on hold
             else:
-                r.stateCd = State.HOLD
+                request.stateCd = State.HOLD
                 event = Event.MARKED_ON_HOLD
 
-            db.session.add(r)
-            EventRecorder.record(user, event, r, r.json(), save_to_session=True)
+            request.save_to_db()
+            EventRecorder.record(user, event, request, request.json(), save_to_session=True)
+
+        # for NRs showing in NRO_UPDATING status need to be set to DRAFT
+        nro_updating_reqs = db.session.query(Request). \
+            filter(Request.stateCd == State.NRO_UPDATING). \
+            order_by(Request.lastUpdate.asc()). \
+            limit(max_rows). \
+            with_for_update().all()
 
-        db.session.commit()
+        for request in nro_updating_reqs:
+            row_count += 1
+            current_app.logger.debug(f'processing nr: {request.nrNum}, state: {request.stateCd}, \
+                previous state: {request.previousStateCd}, last_update: {request.lastUpdate}')
+
+            furnish_request_message(request, 'nro-updating')
```

Comment:
so you are sending a cloud event here to the entity emailer.  is there work planned to update the emailer?  No emails will go out as is as the emailer doesn't know how to deal with the `option` value of `nro-updating`.

Reply:
as per our conversation, emailer corresponding updates will be made as part of another ticket.